### DISCUSSION TITLE
In "wezterm ssh" sessions: how to get the clipboard to the client?

### DISCUSSION THREAD

**Discussion by janlimpens** [wezterm/wezterm#4319] [Q&A]
I open a session `wezterm ssh somehost.com`, open nvim and yank. The clipboard is not exported to my desktop.
In another window, I do just `ssh somehost.com`, nvim, yank, clipboard contents are on my desktop.

Anyone got a solution for this? Probably not a wezterm issue per se, but here are the folks who most likely solved it :)

**Comment by wez** [comment]
I use https://github.com/ojroques/vim-oscyank
